dev-python/markdown2 and dev-python/markdown install their commands
as markdown2 and markdown_py, respectively.

--- markdown-mode-2.4-orig/markdown-mode.el
+++ markdown-mode-2.4/markdown-mode.el
@@ -90,7 +90,7 @@
   :group 'text
   :link '(url-link "https://jblevins.org/projects/markdown-mode/"))
 
-(defcustom markdown-command (let ((command (cl-loop for cmd in '("markdown" "pandoc")
+(defcustom markdown-command (let ((command (cl-loop for cmd in '("markdown" "pandoc" "markdown2" "markdown_py")
                                                     when (executable-find cmd)
                                                     return (file-name-nondirectory it))))
                               (or command "markdown"))
